apps/src/music/analytics/AnalyticsReporter.ts

```diff
@@ -181,6 +182,18 @@ export default class AnalyticsReporter {
     this.log(`Project property: ${property}: ${value}`);
   }
 
+  setSelectedPack(packId: string | undefined) {
+    if (!this.session) {
+      this.log('No session in progress');
+      return;
+    }
+    this.session.selectedPack = packId;
+  }
+
+  onPackSelected(packId: string) {
```

Comment:
@breville as per Slack discussion, created a separate function for reporting pack ID selection that GA4 can also use